tooling/gulpfile.js

```diff
@@ -28,27 +28,26 @@ const rootFolder           = path.join(__dirname, '../');
 
 // General configurations for the build process
 const config = {
-  libName: libName,
+  libName,
   paths: {
-    rootFolder: rootFolder,
+    rootFolder,
     distFolder: path.join(rootFolder, 'dist/terminus-ui'),
     npmFolder: path.join(rootFolder, 'node_modules/@terminus/ui'),
     scss: {
       helpersInputs: [
         path.join(rootFolder, 'terminus-ui', 'scss/helpers/_a11y.scss'),
-        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_typography.scss'),
-        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_cursors.scss'),
-        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_color.scss'),
+        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_animation.scss'),
         path.join(rootFolder, 'terminus-ui', 'scss/helpers/_assets.scss'),
         path.join(rootFolder, 'terminus-ui', 'scss/helpers/_breakpoints.scss'),
+        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_color.scss'),
```

Comment:
buttons
~~card~~

Reply:
I'm not sure what this comment means. Could you clarify?

Reply:
Even though we discussed, I wanted to add a comment in the PR:

I had (incorrectly) assumed all helpers should be exposed and listed out the ones that weren't included.